src/main/java/billgates/entities/IUser.java

```diff
@@ -0,0 +1,5 @@
+package billgates.entities;
+
+public interface IUser {
```

Comment:
Hmm, what does this Interface user contain? 

Usually, an interface should probably have methods that should be overridden by the classes that tries to implment it, in this case, User.java. Maybe try declaring some methods for User to implement?